modules/Hoot/ui/sidebar/sidebar.js

```diff
@@ -168,6 +169,116 @@ export default class Sidebar {
         this.adjustSize();
     }
 
+    saveChanges() {
+        let loadedLayers    = Object.values(Hoot.layers.loadedLayers);
+        let selectReference = d3.selectAll('#reference');
+        let selectSecondary = d3.selectAll('#secondary');
+        if (loadedLayers.length === 2) {
+            let referenceActive = _find(loadedLayers, function(a, b) { return a.refType === 'primary'; });
+            let secondaryActive = _find(loadedLayers, function(a, b) { return a.refType === 'secondary'; });
+            let changeActive    = new LayerAdd();
+
+
+            if (d3.select('#reference button.select-active-layer').empty()) {
```

Comment:
wonder if the on click callback can be generalized in a single function? seems like when we click one fo the active layer buttons, we make the one we click's classes all set to active, and the other one's set to false. so a function could have 2 params, one we want to set to active, one we want to disable and just use those accordingly.

so naively, something like

```javascript
function update(active, inactive) {
   //...update active button class
   //...update inactive button
}
```